npm/plugin/main.go

```diff
@@ -71,17 +73,17 @@ func main() {
 	factor := rand.Float64() + 1
 	resyncPeriod := time.Duration(float64(minResyncPeriod.Nanoseconds()) * factor)
 
-	log.Logf("[INFO] Resync period for NPM pod is set to %d.", int(resyncPeriod/time.Minute))
+	klog.Infof("Resync period for NPM pod is set to %d.", int(resyncPeriod/time.Minute))
 	factory := informers.NewSharedInformerFactory(clientset, resyncPeriod)
 
 	npMgr := npm.NewNetworkPolicyManager(clientset, factory, exec.New(), version)
-	metrics.CreateTelemetryHandle(npMgr.GetAppVersion(), npm.GetAIMetadata())
+	metrics.CreateTelemetryHandle(version, npm.GetAIMetadata())
```

Comment:
Can you fix this lint please, apart from that the PR looks good.

Reply:
As we agreed, it will come with another PR about metric.